### ISSUE TITLE
SQL Committing yet autocommit is active

### ISSUE THREAD

**Issue by erocs** [Civcraft/RealisticBiomes#82]
[00:11:16] [pool-7-thread-1/WARN]: [RealisticBiomes] Can't commit?java.sql.SQLException: Can't call commit when autocommit=true

**Comment by Maxopoly** [comment]
This is safe to ignore. If we turn auto commit of, nothing else will work

**Comment by erocs** [comment]
Yup, can we remove the explicit commit from here to prevent the log spam?
